<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>notIslemleri</class>
 <widget class="QDialog" name="notIslemleri">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1173</width>
    <height>853</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="ogrler">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>431</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="dersler">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>70</y>
     <width>491</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="all">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>1141</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>494</x>
     <y>10</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel { color : red; }</string>
   </property>
   <property name="text">
    <string>Not İşlemleri</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="no">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>380</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Öğrenci No:</string>
   </property>
  </widget>
  <widget class="QLabel" name="ders_kodu">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>440</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ders Kodu:</string>
   </property>
  </widget>
  <widget class="QLabel" name="vize">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>380</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Vize Notu:</string>
   </property>
  </widget>
  <widget class="QLabel" name="final_text">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>440</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Final Notu:</string>
   </property>
  </widget>
  <widget class="QLabel" name="all_text">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>545</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Tüm Öğrenci-Ders Kayıtları</string>
   </property>
  </widget>
  <widget class="QLabel" name="ogr">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>46</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Tüm Öğrenciler</string>
   </property>
  </widget>
  <widget class="QLabel" name="ders">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>45</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Tüm Dersler</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="no_lineEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>380</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="kod_lineEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>440</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="vize_lineEdit">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>380</y>
     <width>271</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="final_lineEdit">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>440</y>
     <width>271</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>490</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ders Kaydı Ekle</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
